core/field_angle.ts

```diff
@@ -21,7 +21,7 @@ import * as Css from './css.js';
 import * as dropDownDiv from './dropdowndiv.js';
 import {Field} from './field.js';
 import * as fieldRegistry from './field_registry.js';
-import {FieldTextInput} from './field_textinput.js';
+import {Config as TextConfig, FieldTextInput} from './field_textinput.js';
```

Comment:
Out of curiosity, why import from `'./field_textinput.js'` instead of `'./field_textinput'`? (Ditto for other imports)

I also think it might be worth renaming each **Config** interface to correspond to the file it belongs with. For example, renaming the **Config** from `'field_textinput'` to always be **TextConfig**. Even though all of the **Config** interfaces correspond to the `opt_config` key, they're each a distinct type of config which could be confusing if a developer is implementing multiple types of fields. (For example, my IDE might tell me at a glance that both `FieldAngle` and `FieldNumber` require **Config** and I'm super confused why I can't pass the same thing into both)

Reply:
> Out of curiosity, why import from './field_textinput.js' instead of './field_textinput'? (Ditto for other imports)

Closure compiler reasons =( If we just use './file' it's not smart enough to find them.

> Re: renamings

Done!